README.md

```diff
@@ -1,87 +1,120 @@
 ![sovrinlogo](https://github.com/sovrin-foundation/sovrin/blob/master/banner.png "insert humor here")
+# Table of Contents
+
+
+1.  [Requirements](#requirements)
+    1.  [Installing Indy-SDK](#installing-indy-sdk)
+2.  [Running Indy Pool](#running-indy-pool)
+    1.  [Build the pool](#build-the-pool)
+    2.  [run the poool](#run-the-poool)
+    3.  [Compiling libsovtoken and running tests](#compiling-libsovtoken-and-running-tests)
+3.  [How To Contribute](#how-to-contribute)
 
 <a href="https://www.apache.org/licenses/LICENSE-2.0.txt" target="_blank">![Hex.pm](https://img.shields.io/hexpm/l/plug.svg?style=plastic)</a>
 <a href="https://badge.fury.io/gh/sovrin-foundation%2Flibsovtoken">[![GitHub version](https://badge.fury.io/gh/sovrin-foundation%2Flibsovtoken.svg)](https://badge.fury.io/gh/sovrin-foundation%2Flibsovtoken)</a>
 # LibSovToken
 
+This is a payment handler library to work with libindy. It may one day
+be merged into libindy.
+
+
+<a id="requirements"></a>
+
+
+# Requirements
+
+-   Rust Lang (Stable)
+-   LibIndy (Stable)
+-   Indy Pool (Provided)
+
 
-This is a payment handler library to work with libindy. It may one day be merged into libindy.
+<a id="installing-indy-sdk"></a>
 
+## Installing Indy-SDK
 
-## Requirements
+1.  Ubuntu
 
-* Rust Lang (Stable)
-* LibIndy (Stable)
-* Indy Pool (Provided)
+        sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68DB5E88
+        sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb xenial stable"
+        sudo apt-get update
+        sudo apt-get install -y libindy>=1.6.0
 
-### Installing Indy-SDK
+2.  macOS
 
-##### Ubuntu
+        git clone https://github.com/hyperledger/indy-sdk.git
+        cd indy-sdk
+        git checkout stable 
+        cd libindy
+        cargo clean 
+        cargo update
+        cargo build
+    
+    Libsovtoken build needs to know how to find Indy-SDK. This is done
+    through the environment variable LIBINDY<sub>DIR</sub>.
```

Comment:
Do you intend this to be LIBINDY<sub>DIR</sub> and not LIBINDY_DIR?

Reply:
This was an exporter error. I will fix it.